app/views/foods/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<!-- app/views/foods/index.html.erb -->
+
+<%= stylesheet_link_tag "foods", "data-turbo-track": "reload" %>
+<h2>Food List</h2>
+<div class="add-food-btn">
+  <%= link_to 'Add Food', new_food_path, class: 'button-white-black' %>
+</div>
+<% if @foods.any? %>
+  <ul class="foods-container">
+    <div class="foods-header">
+      <span>Food</span>
+      <span>Quantity</span>
+      <span>Measurement Unit</span>
+      <span>Unit Price</span>
+      <span>Actions</span>
+    </div>
+    <% @foods.each do |food| %>
+      <% if food.user_id == current_user.id %>
+        <li class="food-record"> 
+          <span><%= food.name %></span>
+          <span><%= food.quantity %></span>
+          <span><%= food.measurement_unit %></span>
+          <span>$ <%= food.price %></span>
+          <%= form_with(model: food, url: food_path(food), method: :delete) do |form| %>
+            <%= form.hidden_field :_method, value: "delete" %>
+            <%= form.submit 'Remove' %>
+          <% end %>
```

Comment:
[REQUIRED] Kindly fix your code. The idea is that anyone who comes to the food list page can see all foods even if they did not create it but they should be able to delete a food only if they created it